packages/swapper/src/swappers/zrx/zrxExecuteTrade/zrxExecuteTrade.test.ts

```diff
@@ -40,7 +40,7 @@ describe('ZrxExecuteTrade', () => {
         chainSpecific: { approvalFee: '123600000', estimatedGas: '1235', gasPrice: '1236' }
       },
       sources: []
-    },
+    } as ZrxTrade<'eip155:1'>,
```

Comment:
please implement this as `const trade: ZrxTrade<'eip155:1> = ...` rather than casting - casting in tests is still dangerous